### ISSUE TITLE
Runtime error: replaceWith a contraction

### ISSUE THREAD

**Issue by derickmelendez** [spencermountain/compromise#744]
Using this code

```
var doc = nlp("The only reason he doesn't continue is because of how tired he feels.");
doc.verbs().toPastTense();
console.log(doc.text());
```

I would expect the following console log:

` The only reason he did not continue is because of how tired he felt`

Actual result is:

`The only reasoned he did not doesn’t`

Error log is:

```
Compromise error: Linked list broken in phrase 'the-yf98lfd'
terms @ compromise.js:1267
matchAll @ compromise.js:2696
match_1 @ compromise.js:2799
lookBehind @ compromise.js:1984
(anonymous) @ compromise.js:6517
exports.lookBehind @ compromise.js:6516
toBe @ compromise.js:12996
conjugate @ compromise.js:13041
(anonymous) @ compromise.js:13228
(anonymous) @ compromise.js:6740
forEach @ compromise.js:6732
toPastTense @ compromise.js:13225
(anonymous) @ contentscript.js:12
compromise.js:1360 Compromise error: Linked list broken. Missing term 'null' in phrase 'the-yf98lfd'

hasId @ compromise.js:1360
(anonymous) @ compromise.js:1614
(anonymous) @ compromise.js:1613
appendPhrase @ compromise.js:1611
replace @ compromise.js:1797
(anonymous) @ compromise.js:7147
replaceWith @ compromise.js:7105
(anonymous) @ compromise.js:13231
(anonymous) @ compromise.js:6740
forEach @ compromise.js:6732
toPastTense @ compromise.js:13225
(anonymous) @ contentscript.js:12
compromise.js:1360 Compromise error: Linked list broken. Missing term 'null' in phrase 'the-yf98lfd'

hasId @ compromise.js:1360
(anonymous) @ compromise.js:1714
(anonymous) @ compromise.js:1713
shrinkAll @ compromise.js:1711
deletePhrase @ compromise.js:1756
replace @ compromise.js:1801
(anonymous) @ compromise.js:7147
replaceWith @ compromise.js:7105
(anonymous) @ compromise.js:13231
(anonymous) @ compromise.js:6740
forEach @ compromise.js:6732
toPastTense @ compromise.js:13225
(anonymous) @ contentscript.js:12
compromise.js:1360 Compromise error: Linked list broken. Missing term 'null' in phrase 'the-yf98lfd'

hasId @ compromise.js:1360
(anonymous) @ compromise.js:1614
(anonymous) @ compromise.js:1613
appendPhrase @ compromise.js:1611
replace @ compromise.js:1797
(anonymous) @ compromise.js:7147
replaceWith @ compromise.js:7105
(anonymous) @ compromise.js:13231
(anonymous) @ compromise.js:6740
forEach @ compromise.js:6732
toPastTense @ compromise.js:13225
(anonymous) @ contentscript.js:12
compromise.js:1360 Compromise error: Linked list broken. Missing term 'null' in phrase 'the-yf98lfd'

hasId @ compromise.js:1360
(anonymous) @ compromise.js:1714
(anonymous) @ compromise.js:1713
shrinkAll @ compromise.js:1711
deletePhrase @ compromise.js:1756
replace @ compromise.js:1801
(anonymous) @ compromise.js:7147
replaceWith @ compromise.js:7105
(anonymous) @ compromise.js:13231
(anonymous) @ compromise.js:6740
forEach @ compromise.js:6732
toPastTense @ compromise.js:13225
(anonymous) @ contentscript.js:12
compromise.js:1360 Compromise error: Linked list broken. Missing term 'null' in phrase 'the-yf98lfd'

hasId @ compromise.js:1360
(anonymous) @ compromise.js:1614
(anonymous) @ compromise.js:1613
appendPhrase @ compromise.js:1611
replace @ compromise.js:1797
(anonymous) @ compromise.js:7147
replaceWith @ compromise.js:7105
(anonymous) @ compromise.js:13231
(anonymous) @ compromise.js:6740
forEach @ compromise.js:6732
toPastTense @ compromise.js:13225
(anonymous) @ contentscript.js:12
compromise.js:1360 Compromise error: Linked list broken. Missing term 'null' in phrase 'the-yf98lfd'

hasId @ compromise.js:1360
(anonymous) @ compromise.js:1714
(anonymous) @ compromise.js:1713
shrinkAll @ compromise.js:1711
deletePhrase @ compromise.js:1756
replace @ compromise.js:1801
(anonymous) @ compromise.js:7147
replaceWith @ compromise.js:7105
(anonymous) @ compromise.js:13231
(anonymous) @ compromise.js:6740
forEach @ compromise.js:6732
toPastTense @ compromise.js:13225
(anonymous) @ contentscript.js:12
compromise.js:1360 Compromise error: Linked list broken. Missing term 'null' in phrase 'the-yf98lfd'

hasId @ compromise.js:1360
(anonymous) @ compromise.js:1614
(anonymous) @ compromise.js:1613
appendPhrase @ compromise.js:1611
replace @ compromise.js:1797
(anonymous) @ compromise.js:7147
replaceWith @ compromise.js:7105
(anonymous) @ compromise.js:13231
(anonymous) @ compromise.js:6740
forEach @ compromise.js:6732
toPastTense @ compromise.js:13225
(anonymous) @ contentscript.js:12
compromise.js:1360 Compromise error: Linked list broken. Missing term 'null' in phrase 'the-yf98lfd'

hasId @ compromise.js:1360
(anonymous) @ compromise.js:1714
(anonymous) @ compromise.js:1713
shrinkAll @ compromise.js:1711
deletePhrase @ compromise.js:1756
replace @ compromise.js:1801
(anonymous) @ compromise.js:7147
replaceWith @ compromise.js:7105
(anonymous) @ compromise.js:13231
(anonymous) @ compromise.js:6740
forEach @ compromise.js:6732
toPastTense @ compromise.js:13225
(anonymous) @ contentscript.js:12
compromise.js:1267 Compromise error: Linked list broken in phrase 'the-yf98lfd'

```

**Comment by spencermountain** [comment]
whoa!
can confirm. This is bad. Thank you for the heads-up.

**Comment by spencermountain** [comment]
the verb conjugation is creating a contraction, which isn't getting stitched-back into the term-list properly:
```js
let doc = nlp('will go foobar')
doc.replace('will', "before i'm")
console.log(doc.text())
```

**Comment by derickmelendez** [comment]
Glad to help and thanks for taking a look at this.

**Comment by spencermountain** [comment]
fixed in `13.2.0`
```js
doc = nlp("The only reason he doesn't continue is because of how tired he feels.")
doc.verbs().toPastTense()
doc.text()
// 'The only reason he did not continue was because of how tired he felt.'
```
👍
